parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -147,6 +147,31 @@ installCredentalProvider() {
     rm -rf ${CREDENTIAL_PROVIDER_DOWNLOAD_DIR}
 }
 
+# TODO (alburgess) have oras version managed by dependant or Renovate
+installOras() {
+    ORAS_VERSION="1.2.0"
```

Comment:
We should move the download URL and version to components.json (as a package) instead of hard coding it here. There are some examples you can refer to. Let me know if you encounter any issue.